### ISSUE TITLE
Docs for Platform Customization should mention Merges

### ISSUE THREAD

**Issue by cfjedimaster** [driftyco/ionic#4302]
The docs here (http://ionicframework.com/docs/platform-customization/) discuss how Ionic lets you do customizations per platform. While it isn't Ionic-specific, I think it would be useful to mention the Merges feature here and link to the Cordova doc. (There isn't a great URL for that, but it can be found here: http://cordova.apache.org/docs/en/5.1.1/guide_cli_index.md.html#The%20Command-Line%20Interface)

**Comment by mhartington** [comment]
@cfjedimaster Yes merges can be used, but it seems like it's take a bit of a back seat if I recall. Even creating a blank project with the cordova cli, they seemed to have removed the merges folder. Though the functionality is still there, is it going to be removed?

**Comment by cfjedimaster** [comment]
I haven't heard anything about it being removed. I kinda think its cool in
that you don't modify your code to handle the different platforms, it just
happens.

But I'll ask on the Cordova dev list.

On Tue, Sep 1, 2015 at 10:41 AM, Mike Hartington <notifications@github.com>
wrote:

> @cfjedimaster <https://github.com/cfjedimaster> Yes merges can be used,
> but it seems like it's take a bit of a back seat if I recall. Even creating
> a blank project with the cordova cli, they seemed to have removed the
> merges folder. Though the functionality is still there, is it going to be
> removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/driftyco/ionic/issues/4302#issuecomment-136764639>.
>



-- 
===========================================================================
Raymond Camden, Developer Advocate for MobileFirst at IBM

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden

**Comment by cfjedimaster** [comment]
I asked - no official plans to deprecate.

To be clear, I know it isn't a very well known feature, but I think a small
mention would be nice. ;)

On Tue, Sep 1, 2015 at 10:42 AM, Raymond Camden <raymondcamden@gmail.com>
wrote:

> I haven't heard anything about it being removed. I kinda think its cool in
> that you don't modify your code to handle the different platforms, it just
> happens.
>
> But I'll ask on the Cordova dev list.
>
> On Tue, Sep 1, 2015 at 10:41 AM, Mike Hartington <notifications@github.com
> > wrote:
>
>> @cfjedimaster <https://github.com/cfjedimaster> Yes merges can be used,
>> but it seems like it's take a bit of a back seat if I recall. Even creating
>> a blank project with the cordova cli, they seemed to have removed the
>> merges folder. Though the functionality is still there, is it going to be
>> removed?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/driftyco/ionic/issues/4302#issuecomment-136764639>.
>>
>
>
>
> --
> ===========================================================================
> Raymond Camden, Developer Advocate for MobileFirst at IBM
>
> Email : raymondcamden@gmail.com
> Blog : www.raymondcamden.com
> Twitter: raymondcamden
>



-- 
===========================================================================
Raymond Camden, Developer Advocate for MobileFirst at IBM

Email : raymondcamden@gmail.com
Blog : www.raymondcamden.com
Twitter: raymondcamden

**Comment by mhartington** [comment]
Sounds good, I'm moving this to ionic-site repo for now. The right place for this issue.
